src/acquisition/covid_hosp/common/database.py

```diff
@@ -159,6 +159,26 @@ def insert_metadata(self, publication_date, revision, meta_json, logger=False):
           (%s, %s, %s, %s, %s, NOW())
       ''', (self.table_name, self.hhs_dataset_id, publication_date, revision, meta_json))
 
+  def remove_issues(self, issue_date):
+    # TODO: this is *VERY* incomplete!  SQL statements are never even evaluated!
+    # delete from metadata table where issue date matches
+    a = f"DELETE FROM `covid_hosp_meta` WHERE dataset_name='{self.table_name}' AND publication_date='{issue_date}'"
+    if self.aggregate_key_cols:
+      # TODO: restrict this to just UNIQUE columns from aggregate keys table?
+      # create (empty) `some_temp_table` like `{self.table_name}_key`
+      b = f"CREATE TABLE some_temp_table AS SELECT {self.aggregate_key_cols} FROM `{self.table_name}_key` WHERE FALSE"
+      # save aggregate keys from what we are about to delete
+      c = f"SELECT {self.aggregate_key_cols} INTO some_temp_table FROM `{self.table_name}` WHERE `{self.publication_col_name}`={issue_date} GROUP BY {self.aggregate_key_cols}"
```

Comment:
```suggestion
      c = f"INSERT {self.aggregate_key_cols} INTO some_temp_table FROM `{self.table_name}` WHERE `{self.publication_col_name}`={issue_date}"
```

Should this be INSERT INTO? [SELECT INTO](https://dev.mysql.com/doc/refman/8.0/en/select-into.html) seems to only store to vars or files. Not sure if we need the GROUP BY either.